posts/2022-08-01-Increasing-glibc-kernel-requirements.md

```diff
@@ -0,0 +1,116 @@
+---
+layout: post
+title: "Increasing the glibc and kernel requirements"
+author: Nikita Popov
+---
+
+The minimum requirements for Rust toolchains targeting Linux will [increase][PR] with the
+Rust 1.64.0 release (slated for September 22nd, 2022). The new minimum requirements are:
+
+* glibc >= 2.17 (previously glibc >= 2.11)
+* kernel >= 3.2 (previously kernel >= 2.6.32)
+
+These requirements apply both to running the Rust compiler itself (and other Rust tooling like
+Cargo or Rustup), and to running binaries produced by Rust, if they use the libstd.
+
+If you are not targeting an old long-term-support distribution, or embedded hardware running
+an old Linux version, this change is unlikely to affect you. Otherwise, read on!
+
+# Affected targets
+
+In principle, the new kernel requirements affect all `*-linux-*` targets, while the glibc
+requirements affect all `*-linux-gnu*` targets. In practice, many targets were already requiring
+newer kernel or glibc versions. The requirements for such targets do not change.
+
+Among targets for which a Rust host toolchain is distributed, the following *are* affected:
+
+* `i686-unknown-linux-gnu` (Tier 1)
```

Comment:
Since you're mentioning tiers, we should probably link to the platform support page. In release notes, we usually copy a side note with the following boiler-plate text, but feel free to write something in your own voice.

> Refer to Rust's [platform support page](https://doc.rust-lang.org/nightly/rustc/platform-support.html) for more information on Rust's tiered platform support.

Reply:
I've added a link to the platform support page, together with a note that the requirements listed there aren't up to date yet. (I'll try to remember to go back and drop this once the docs get updated.)